util/issue_commands.go

```diff
@@ -14,9 +14,15 @@ func InitRDDLReissuanceProcess(ctx sdk.Context, proposerAddress string, tx_unsig
 	//get_last_PoPBlockHeight() // TODO: to be read form the upcoming PoP-store
 	// Construct the command
 	sending_validator_address := config.GetConfig().ValidatorAddress
+	keyring := config.GetConfig().PlanetmintKeyring
 	cmd := exec.Command("planetmint-god", "tx", "dao", "reissue-rddl-proposal",
 		"--from", sending_validator_address, "-y",
 		proposerAddress, tx_unsigned, strconv.FormatInt(blk_height, 10))
+	if keyring != "" {
+		cmd = exec.Command("planetmint-god", "tx", "dao", "reissue-rddl-proposal",
```

Comment:
Better to do it similarly like @LaurentMontBlanc does it to not copy&paste too much
```
	cmdStr := fmt.Sprintf("%s tx dao mint-token '%s' --from %s -y", planetmint, string(mrJSON), planetmintAddress)

	if planetmintKeyring != "" {
		cmdStr = fmt.Sprintf("%s --keyring-backend %s", cmdStr, planetmintKeyring)
	}
```
Please change in both places